index.html

```diff
@@ -0,0 +1,177 @@
+                    <div class="mindetails">
+                        <a class="mintitle" href="#">Boku no Hero</a>
+                        <ul class="mininfo">
+                            <li>Uploaded by</li>
+                            <li>Visualization</li>
+                            <li>Date</li>
+                        </ul>
+                    </div>
+                </li>
+                <li class="vidinfo"><img class="minimages" src="Images/images (5).jpeg" alt="img taken from google">
+                    <div class="mindetails">
+                        <a class="mintitle" href="#">Atack on titan</a>
+                        <ul class="mininfo">
+                            <li>Uploaded by</li>
+                            <li>Visualization</li>
+                            <li>Date</li>
+                        </ul>
+                    </div>
+                </li>
+                <li class="vidinfo"><img class="minimages" src="Images/images (6).jpeg" alt="img taken from google">
+                    <div class="mindetails">
+                        <a class="mintitle" href="#">Anime legacy</a>
+                        <ul class="mininfo">
+                            <li>Uploaded by</li>
+                            <li>Visualization</li>
+                            <li>Date</li>
+                        </ul>
+                    </div>
+                </li>
+                <li class="vidinfo"><img class="minimages" src="Images/images (7).jpeg" alt="img taken from google">
+                    <div class="mindetails">
+                        <a class="mintitle" href="#">Demon Slayer</a>
+                        <ul class="mininfo">
+                            <li>Uploaded by</li>
+                            <li>Visualization</li>
+                            <li>Date</li>
+                        </ul>
+                    </div>
+                </li>
+                <li class="vidinfo"><img class="minimages" src="Images/images (8).jpeg" alt="img taken from google">
+                    <div class="mindetails">
+                        <a class="mintitle" href="#">Evil eye</a>
+                        <ul class="mininfo">
+                            <li>Uploaded by</li>
+                            <li>Visualization</li>
+                            <li>Date</li>
+                        </ul>
+                    </div>
+                </li>
+                <li class="vidinfo"><img class="minimages" src="Images/images (9).jpeg" alt="img taken from google">
+                    <div class="mindetails">
+                        <a class="mintitle" href="#">Hime</a>
+                        <ul class="mininfo">
+                            <li>Uploaded by</li>
+                            <li>Visualization</li>
+                            <li>Date</li>
+                        </ul>
+                    </div>
+                </li>
+            </ul>
+
+        </aside>
+
+    </div>
+    
+</body>
+</html>
```

Comment:
Please add an empty line at the end of your code. This is again for code semantic purposes.